jsonrpc/server.go

```diff
@@ -71,7 +71,7 @@ func Err(code int, data any) *Error {
 	case InvalidParams:
 		return &Error{Code: InvalidParams, Message: "Invalid Params", Data: data}
 	default:
-		return &Error{Code: InternalError, Message: "Internal Error", Data: data}
+		return &Error{Code: InternalError, Message: "Internal error", Data: data}
```

Comment:
"error" it the [spec](https://www.jsonrpc.org/specification#:~:text=%2D32603-,Internal%20error,-Internal%20JSON%2DRPC) + `rpc.InternalError` message already lowercased